src/components/shared/Entity/DetailsForm.tsx

```diff
@@ -69,7 +112,13 @@ function DetailsForm({
                 },
             });
         }
-    }, [connectorID, setDetails]);
+    }, [setDetails, connectorID]);
+
+    useEffect(() => {
```

Comment:
Previously, the `setDetails` action reset the form state. With the details form and form state logic in separate stores, the reset form state logic was migrated to this effect.